api/src/main/java/org/pmiops/workbench/api/BigQueryService.java

```diff
@@ -151,6 +158,6 @@ public FieldList getTableFieldsFromDomain(Domain d) {
     TableId tableId =
         TableId.of(cdrVersion.getBigqueryProject(), cdrVersion.getBigqueryDataset(), tableName);
 
-    return bigquery.getTable(tableId).getDefinition().getSchema().getFields();
+    return getBigQueryService().getTable(tableId).getDefinition().getSchema().getFields();
```

Comment:
What do you want to happen if the table with id `tableId` isn't found? I don't think that will map to an empty field list.